backend/src/main/java/com/karankumar/bookproject/backend/service/UserService.java

```diff
@@ -137,6 +147,14 @@ public boolean emailIsNotInUse(String email) {
     }
 
     public void changeUserPassword(@NonNull User user, @NonNull String password) {
+        Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
+        Set<ConstraintViolation<String>> constraintViolations = validator.validate(password);
```

Comment:
This line and the line above can be removed. We use it for testing the constraints we place on the `user` entity (see the annotations like `@Email` and password strength in User.java).